docs/SETUP_PROVISIONING.md

```diff
@@ -0,0 +1,698 @@
+# Provisioning of infrastructure on AWS
+
+It is assumed you have read through [docs/SETUP_CI_CD_PIPELINE.md](SETUP_CI_CD_PIPELINE.md) and prepared the Gitlab pipeline as documented.
+The first step is to connect and prepare to your AWS account from which the resources of the infrastructure to provision
+will be associated with.
+If you don't have an account yet, ask a core team member and they will do so 
+and also generate the domain names you will need later to access certain endpoints of the infrastructure.
+
+## AWS dashboard
+
+### Choosing a region
+
+The very first step is to choose a region where to deploy your infrastructure.
+It has to be a region not used by someone else, that has the specific AWS AMI we need for our EC2 instances, 
+and finally it should be authorised in our policies to be used. Below is a table showing you the choices.
+Once you pick an unused region, we will update this document to mark your name on your choice.
+
+| Region code | Region name | Used by |  Comments |
+| --- | --- | --- | --- |
+| ap-east-1 | Hong Kong | gigadb | |
+|  ap-northeast-1 |  Tokyo | Peter | |
+|  ap-southeast-1 | Singapore | gigabyte, gigablog | |
+|  ap-southeast-2 |  Sidney | gigadb (alternative) | |
+| ap-southeast-4 | Melbourne | | Under preparation, available soon |
+| ap-northeast-2 | Osaka | | Don't use, it doesn't have Centos 8 Stream X86_64 |
+| ap-northeast-3 | Seoul | |Don't use, it doesn't have Centos 8 Stream X86_64|
```

Comment:
Hi @rija:

`ap-northeast-2` is used by me, and its region is Seoul.
`ap-northeast-3` region should be Osaka.

Details at here: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html

And regarding the current ami owner `125523088429` we are using, who does provide `Centos 8 Stream X86_64` ami in the Seoul region as you can see from this query: https://ap-northeast-2.console.aws.amazon.com/ec2/home?region=ap-northeast-2#Images:visibility=public-images;owner=125523088429;imageName=:CentOS%20Stream%208%20x86_64;v=3;$case=tags:false%5C,client:false;$regex=tags:false%5C,client:false

But the `Centos 8 Stream X86_64` ami  is unavailable in the Osaka (ap-northeast-3) region.

FYI, I was able to deploy alternative gigadb website from Europe (Stockholm) `eu-north-1`.  

In fact, I was wondering is it necessary to mention `Don't use, ...` in the comments? If the region cannot be used, then is it better not to mention it in the doc?

Reply:
Thanks @kencho51, 
I've updated the table accordingly.
I've also removed Spain and Melbourne for now as they don't have the AMI we use currently.